test/jdk/java/nio/channels/FileChannel/BlockDeviceSize.java

```diff
@@ -34,31 +34,40 @@
 import java.nio.channels.FileChannel;
 import java.nio.file.AccessDeniedException;
 import java.nio.file.NoSuchFileException;
+import java.util.List;
+
 import static java.nio.file.StandardOpenOption.*;
 
 
 public class BlockDeviceSize {
-    private static final String BLK_FNAME = "/dev/sda1";
-    private static final Path BLK_PATH = Paths.get(BLK_FNAME);
+    private static final List<String> BLK_FNAMES = List.of("/dev/sda1", "/dev/nvme0n1") ;
+
+    public static Path getBlkPath(String blkFileName) {
+        return Paths.get(blkFileName);
+    }
 
     public static void main(String[] args) throws Throwable {
-        try (FileChannel ch = FileChannel.open(BLK_PATH, READ);
-             RandomAccessFile file = new RandomAccessFile(BLK_FNAME, "r")) {
+        for (String blkFname: BLK_FNAMES) {
+            Path blkPath = getBlkPath(blkFname);
```

Comment:
You can replace this with Path.of(blkFname), no need for the getBlkPath method.

Reply:
Indeed, it seems unnecessary to create a method only for that. I've included your suggestion with 08744f4262c7b63b25c7c4e65c430948447f711f.